annotated.go

```diff
@@ -335,6 +439,13 @@ func Annotate(f interface{}, anns ...Annotation) interface{} {
 		}
 	}
 
+	if e := annotations.genAnnotatedOutStruct(); e != nil {
+		return annotationError{
+			target: f,
+			err:    e,
```

Comment:
error variables are typically named err unless there's another err in scope that also has to be valid.
```suggestion
	if err := annotations.genAnnotatedOutStruct(); err != nil {
		return annotationError{
			target: f,
			err:    err,
```